### DISCUSSION TITLE
第3节课作业(14班)

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#177] [General]
14班作业～～～

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135500778

**Comment by tianyabanbu** [comment]
https://qiz21j4rsiq.feishu.cn/docx/HKpMdjBdYoTVKsxWmzycRzsLnjd?from=from_copylink

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/vviwapmcx7o5bfua?singleDoc# 《知识库搭建》

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135546553?csdn_share_tail   笔记+作业

**Comment by xilimala** [comment]
https://zhuanlan.zhihu.com/p/677319771

**Comment by ZZfive** [comment]
https://github.com/InternLM/tutorial/discussions/177

**Comment by ucas-ai** [comment]
https://blog.csdn.net/u012468965/article/details/135594234?spm=1001.2014.3001.5502

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/DAVwdN3QLof6HsxPxvjcSdmznQg?from=from_copylink

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/task/third%20task.md

**Comment by wakefuldreamer** [comment]
http://t.csdnimg.cn/0OaJk

**Comment by lizihe11** [comment]
您好！您的邮件已收到！我查阅后，尽快回复您！
